### DISCUSSION TITLE
How to hide apps

### DISCUSSION THREAD

**Discussion by CorruptionVR** [EmiyaSyahriel/CrossLauncher#88] [Q&A] [answered]
Hey, how do I hide apps? I see that it is a completed feature, but how do I hide apps?

**Comment by EmiyaSyahriel** [comment, marked as answer]
- Navigate to the app
- Open Side Menu (by pressing Xbox Y/PS Triangle/Android Remote Menu or by doing left swipe at the right-hand side of screen)
- Navigate to `App Info` (on touch screen, tap upside and downside of the menu to navigate, tap center of it to confirm)
- On the App Info screen, navigate to `Hidden` (on touch screen, navigate by up/down swipe gesture)
- Press or Tap on `Toggle` button to change it to `Yes`
- Press or Tap on `Back`, the app will be hidden

To reveal the app back, go to `Settings > System Settings > Show Hidden Apps` (kept until launcher restarted or changed back to `No`)

**Reply by CorruptionVR** [reply]
oh, ok thanks
